Resources/Private/Solr/configsets/ext_solr_11_1_0/conf/danish/schema.xml

```diff
@@ -104,6 +111,13 @@
 			<filter class="solr.ScandinavianNormalizationFilterFactory"/>
 
 			<filter class="solr.ManagedSynonymGraphFilterFactory" managed="${solr.core.name}" />
+			<filter class="solr.DictionaryCompoundWordTokenFilterFactory"
+					dictionary="danish/danish-common-nouns.txt"
+					minWordSize="5"
+					minSubwordSize="4"
+					maxSubwordSize="15"
+					onlyLongestMatch="false"
+			/>
 			<filter class="solr.FlattenGraphFilterFactory"/>
```

Comment:
Same as above "DictionaryCompoundWordTokenFilterFactory after FlattenGraphFilterFactory"

Reply:
Yes, so I just pushed (with force) fixed commit. I don't know if it is right thing as it is my first pull request. Maybe I should just add new commit with corrections??